environments/production/app-processes.yml

```diff
@@ -36,14 +34,12 @@ celery_processes:
       num_workers: 3
     case_rule_queue:
       concurrency: 4
-      max_tasks_per_child: 1
```

Comment:
A low value like this looks like it might have been set to deal with memory leaks or general bad behavior when multiple tasks are run. Seems like the ones that were `1` may be worth keeping an eye on.

Reply:
👍